### PR TITLE
[Backup] Add support to specify ResourceGroup for storing instantRP during Create or Modify Policy 

### PR DESCRIPTION
**Pull request by sambitratha**
**Description**  
Added support to specify ResourceGroup for stroing instantRP during Create or Modify Policy 

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

File: src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py [hunk 1]
```diff
@@ -207,6 +209,7 @@ def test_backup_policy(self, resource_group, vault_name, policy1, policy2, vm1,
         ])
 
         self.kwargs['policy1_json']['name'] = self.kwargs['policy3']
+        self.kwargs['policy1_json']['properties']['instantRpDetails'] = {'azureBackupRgNamePrefix' : 'RG_prefix', 'azureBackupRgNameSuffix': 'RG_suffix'}
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py:212]: Will this property be in the command result? If so, we should add a check for it.

Comment (sambitratha) [reply src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py]: Added a check

**Comment by fengzhou-msft** [comment]
Is `stroring` in title a typo?

**Comment by sambitratha** [comment]
> Is `stroring` in title a typo?

Yes. Changed the PR title.

**Pull request merged**